xml/Microsoft.Extensions.Options/ValidateOptionsResult.xml

```diff
@@ -121,6 +121,32 @@
         <AssemblyVersion>6.0.0.0</AssemblyVersion>
         <AssemblyVersion>7.0.0.0</AssemblyVersion>
       </AssemblyInfo>
+      <Attributes>
+        <Attribute FrameworkAlternate="dotnet-plat-ext-7.0">
+          <AttributeName Language="C#">[System.Diagnostics.CodeAnalysis.MemberNotNullWhen(true, "Failures")]</AttributeName>
```

Comment:
@huangmin-ms These annotations seem excessive - there are 3 for each annotated property. See https://github.com/dotnet/runtime/blob/c74a3fdf01d57a14218056475f952aae4b6f4e2e/src/libraries/Microsoft.Extensions.Options/src/ValidateOptionsResult.cs#L38-L40 for source code.